app/components/agenda/agendaitem/create-agendaitem.hbs

```diff
@@ -24,77 +14,99 @@
                 <Auk::Input
                   @icon="search"
                   @block={{true}}
-                  @value={{this.filterParam}}
+                  @value={{this.filter}}
                   id="searchId"
                   placeholder={{t "search-placeholder"}}
-                  disabled={{this.showPostponed}}
                   data-test-create-agendaitem-input={{true}}
-                  {{on "input" (perform this.searchTask)}}
+                  {{on "input" this.searchTask.perform}}
```

Comment:
Why not using `(perform this.searchTask)`?

Reply:
not sure why :brain: